sp800-63-3/sec5_xAL.md

```diff
@@ -71,9 +73,7 @@ This guideline introduces a model where individual xALs can be selected without
 | 3 | 2 | 2 | 2
 | 4 | 3 | 3 | 3
 
-In the above table, note: 
-* LOA2 requirements are now equivalent to LOA3.
-* Higher xALs than specified can always be used.
+> Note: LOA2 requirements are now equivalent to LOA3.
```

Comment:
Mike expressed a slight preference for the space so that was enough to push me that direction.